pyproject.toml

```diff
@@ -1,3 +1,64 @@
+[tool.poetry]
+name = "pelorus"
+version = "0.1.0" # we are not using it yet
+description = "Automate the measurement of organizational behavior"
+authors = [
+    "etsauer",
+    "KevinMGranger",
+    "mateusoliveira43",
+    "mpryc",
+    "weshayutin"
+]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+asyncio = "^3.4.3"
+attrs = "^23.1.0"
+azure-devops = "6.0.0b4"
+exceptiongroup = "^1.1.1"
+fastapi = "^0.95.1"
+git-url-parse = "^1.2.2"
+jira = "3.4.1"
+jsonpath-ng = "^1.5.3"
+openshift = "0.13.1"
+prometheus-client = "^0.16.0"
+pygithub = "^1.58.1"
+python-gitlab = ">=2.4.0" # change to ^?
```

Comment:
I think here is the discussion about how to pin the dependencies

For example, we are using version 3 of `python-gitlab` (check `poetry.lock` or `exporters/requirements.txt` files), but we tell here that we accept `2.4` or above, but we do not test that (our CI and we will install version pined in `poetry.lock` or `exporters/requirements.txt` files). I think is best to us use `^` (indicates that we only allow higher minor or patch versions, no major, that could break the code)

More about pining versions in Poetry https://python-poetry.org/docs/dependency-specification/#version-constraints